puppet/zulip/manifests/thumbor.pp

```diff
@@ -0,0 +1,40 @@
+# Class: zulip::thumbor
+#
+#
+class zulip::thumbor {
```

Comment:
This module should `include zulip::supervisor` and `include zulip::nginx`, since it in practice depends on them.